packages/docs/components/Playground.tsx

```diff
@@ -15,6 +15,7 @@ export default ({ mode }: {mode?: 'js' | 'node'}) => {
   return (
     <div className="w-full" style={{ height: 'calc(100vh - 72px)' }} data-cy="playground">
       <DataStory
+        clientv2={new JsClientV2()}
         slotComponents={[
```

Comment:
Whenever the parent component re-renders, `new JsClientV2()` ends up creating a new instance of `JsClientV2`. I recommend using `useCreation` from ahooks to prevent unnecessary re-renders of the component.